data/experiment-recipe-samples/ios-39-multifeature.json

```diff
@@ -0,0 +1,87 @@
+{
+  "appId": "org.mozilla.ios.Firefox",
+  "appName": "firefox_ios",
+  "application": "org.mozilla.ios.Firefox",
+  "arguments": {},
+  "branches": [
+    {
+      "features": [
+        {
+          "enabled": true,
+          "featureId": "onboarding-default-browser",
+          "value": {
+            "should-hide-title-image": false
+          }
+        },
+        {
+          "enabled": true,
+          "featureId": "homescreen",
+          "value": {
+            "sections-enabled": {
+              "jumpBackIn": false,
+              "libraryShortcuts": true,
+              "recentlySaved": false
+            }
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "control"
+    },
+    {
+      "features": [
+        {
+          "enabled": true,
+          "featureId": "onboarding-default-browser",
+          "value": {
+            "should-hide-title-image": true
+          }
+        },
+        {
+          "enabled": true,
+          "featureId": "homescreen",
+          "value": {
+            "sections-enabled": {
+              "jumpBackIn": true,
+              "libraryShortcuts": false,
+              "recentlySaved": true
+            }
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "treatment"
+    }
+  ],
+  "bucketConfig": {
+    "count": 10000,
+    "namespace": "firefox-ios-multifeature-test",
+    "randomizationUnit": "nimbus_id",
+    "start": 0,
+    "total": 10000
+  },
+  "channel": "release",
+  "endDate": null,
+  "featureIds": [
+    "onboarding-default-browser",
+    "homescreen"
+  ],
+  "id": "firefox-ios-multifeature-test",
+  "isEnrollmentPaused": true,
```

Comment:
Same as for the others, having enrollment not paused might help test experiment interaction easier.